src/benchmarks/micro/corefx/System.Linq/Perf.Linq.cs

```diff
@@ -107,54 +114,64 @@ public void Cast_SameType(int size, int iteration, Perf_LinqTestBase.WrapperType
+        public void OrderByDescending(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
             => Perf_LinqTestBase.Measure(_sizeToPreallocatedArray[size], wrapType, col => col.OrderByDescending(o => -o), _consumer);
 
         [Benchmark]
         [ArgumentsSource(nameof(IterationSizeWrapperData))]
-        public void OrderByThenBy(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType) 
+        public void OrderByThenBy(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
             => Perf_LinqTestBase.Measure(_sizeToPreallocatedArray[size], wrapType, col => col.OrderBy(o => -o).ThenBy(o => o), _consumer);
 
         [Benchmark]
         [Arguments(DefaultSize, DefaulIterationCount)]
-        public void Range(int size, int iteration) 
+        public void Range(int size, int iteration)
             => Enumerable.Range(0, size).Consume(_consumer);
 
         [Benchmark]
         [Arguments(DefaultSize, DefaulIterationCount)]
-        public void Repeat(int size, int iteration) 
+        public void Repeat(int size, int iteration)
             => Enumerable.Repeat(0, size).Consume(_consumer);
 
         [Benchmark]
         [ArgumentsSource(nameof(IterationSizeWrapperData))]
-        public void Reverse(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType) 
+        public void Reverse(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
             => Perf_LinqTestBase.Measure(_sizeToPreallocatedArray[size], wrapType, col => col.Reverse(), _consumer);
 
         [Benchmark]
         [ArgumentsSource(nameof(IterationSizeWrapperData))]
-        public void Skip(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType) 
+        public void Skip(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
             => Perf_LinqTestBase.Measure(_sizeToPreallocatedArray[size], wrapType, col => col.Skip(1), _consumer);
 
         [Benchmark]
         [ArgumentsSource(nameof(IterationSizeWrapperData))]
-        public void Take(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType) 
+        public void Take(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
             => Perf_LinqTestBase.Measure(_sizeToPreallocatedArray[size], wrapType, col => col.Take(size - 1), _consumer);
 
 #if !NETFRAMEWORK
         [Benchmark]
-        [ArgumentsSource(nameof(IterationSizeWrapperData))]
+        [ArgumentsSource(nameof(IterationMultipleSizesWrapperData))]
+        public void TakeLastOne(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
+            => Perf_LinqTestBase.Measure(_multipleSizesToPreallocatedArray[size], wrapType, col => col.TakeLast(1), _consumer);
+
+        [Benchmark]
+        [ArgumentsSource(nameof(IterationMultipleSizesWrapperData))]
         public void TakeLast(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
```

Comment:
to follow the convention you have introduced the method should be named TakeLast**Half**

```suggestion
        public void TakeLastHalf(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType)
```

Reply:
Yeah, this looks like a typo left from one of my numerous versions of the code.